DotNet/LinkAdmin.BFF/Program.cs

```diff
@@ -42,6 +42,11 @@
 #region Register Services
 static void RegisterServices(WebApplicationBuilder builder)
 {
+
+    //Initialize activity source
+    var version = Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion ?? string.Empty;
```

Comment:
Should this be applied to all services? I only see this change in `Account/Program.cs` right now...

Reply:
I only implemented it in LinkAdmin for now to get feedback on the approach. I guess I probably should have removed it or just added it to all the services.